crates/pop-parachains/src/errors.rs

```diff
@@ -11,6 +11,8 @@ pub enum Error {
 	AnyhowError(#[from] anyhow::Error),
 	#[error("Failed to establish a connection to the API: {0}")]
 	ApiConnectionFailure(String),
+	#[error("Failed to decode call data. {0}")]
```

Comment:
Missing doc comment - lets try to ensure that we have them for any new variants so we can slowly get full doc coverage for errors.